src/components/Header.tsx

```diff
@@ -106,7 +108,16 @@ const HeaderLinks = () => {
 const Header = (props: BoxProps) => {
   const isClientSide = useIsClientSide()
   return (
-    <Box as="header" bg="gray.700" color="white" zIndex="banner" {...props}>
+    <Box
+      as="header"
+      zIndex="banner"
+      {...props}
+      borderTopWidth={{ base: '4px', sm: '8px' }}
+      borderColor="fuchsia.700"
+      // TODO: Remove borderBottomWidth and borderBottomColor once we fix the issue
+      borderBottomWidth="8px"
+      borderBottomColor="sky.700"
```

Comment:
I've added some properties to help visualize the problem.
I've added a big line under the header.

I notice that the `sky.700` line jumps around when you resize the window.

Reply:
https://user-images.githubusercontent.com/2677072/125446140-928e8185-7a43-4480-ac3e-5d7f3de1eb49.mov

I guess this is the "problem" you mentioned? If so, that's just because the Header text is changing size between mobile, desktop, and tablet. That happened already, we can live with it for now.

If you mean something else, I'm absolutely lost.

Reply:
I think the Octoclairvoyant image resize is messing with this a bit.

Ideally I'd want the baseline for the text of the Octoclairvoyant and the Comparator/About links to be on the same horizontal line.

https://en.wikipedia.org/wiki/Baseline_(typography)

---

But maybe I'm worrying too much over a temporary design...
We'll update this component later, so that it conforms to the Figma design.

I'd say we remove the debug code, review one last time, and merge it, and be done with it for now.
The header and drawer still look way better than what we have now.
That way we can focus our attention on the other items on our project board. 😉

Reply:
It might be the logo being resized too, but I would ignore it for now and fix it in a later step of the revamping process.